### PR TITLE
New action for Physics behavior: Set the linear velocity towards an angle

### PR DESCRIPTION
**Pull request by tristanbob**
This currently is not working.  Can someone help me troubleshoot this?

Here is the event:

Screenshot 2023-09-19 150500

...and the error output: 

Screenshot 2023-09-19 150542

### CODE REVIEW

**Comment by tristanbob** [comment]
I think this problem is very simple, can I get a hint?
I recompiled GDevelop.js but it did not help.  
Here is the compiled runtime javascript:

image

**Comment by tristanbob** [comment]
Bump.  Can someone help me with this?  It's very simple and small, but not working.  I must be overlooking something.

**Comment by 4ian** [comment]
Can you share the project? I don't see any function like this in the current codebase. https://github.com/search?q=repo%3A4ian%2FGDevelop%20setLinearVelocityAngle&type=code

**Comment by tristanbob** [comment]
> Can you share the project? I don't see any function like this in the current codebase. https://github.com/search?q=repo%3A4ian%2FGDevelop%20setLinearVelocityAngle&type=code

Thanks for your response.  I have attached the project. It's a single sprite with physics behavior and a single action to set the linear velocity of the object.

[set-linear-velocity-game.zip](https://github.com/4ian/GDevelop/files/13762984/set-linear-velocity-game.zip)

You probably could not find it with your search because the branch is on my repo:
https://github.com/search?q=repo%3Atristanbob%2FGDevelop++setLinearVelocityAngle&type=code

**Comment by tristanbob** [comment]
The commit that includes this function is here:

https://github.com/4ian/GDevelop/pull/5670/commits/f19d8ceccb826313136d9a5f08d5e865a744969a

**Comment by tristanbob** [comment]
I would like to use this feature in a game I'm building, and it reminded me that this feature is not added yet.
I'll try to review it again tomorrow to see if I can get it to pass tests.

**Comment by tristanbob** [comment]
It's working!!!

Reason it didn't work?  We MUST run the electron app at least once.

https://github.com/4ian/GDevelop/assets/8879811/90836211-26ff-4513-ae28-5fd1c92322e9

More context:
https://twitter.com/VictrisGames/status/1770625683603259765

**Comment by tristanbob** [comment]
This PR is ready to be reviewed.

File: newIDE/electron-app/yarn.lock
Comment (AlexandreSi) [inline newIDE/electron-app/yarn.lock:1]: It's weird you had to update this. I guess you used `yarn` to install node dependencies?

**Review by AlexandreSi — APPROVED** [review]
LGTM but I think the changes on yarn.lock's should be discarded

File: newIDE/electron-app/yarn.lock
Comment (tristanbob) [reply newIDE/electron-app/yarn.lock:1]: No, I used NPM. I don't know why this changed, perhaps by the NPM format script?

**Comment by tristanbob** [comment]
I reverted the two commits I had made after running "npm format" in two different directories. I guess those weren't needed!
Now, the PR only includes the changes I wanted!  
I tested it after these changes and it worked great.
Thanks for your help!

**Comment by AlexandreSi** [comment]
Thanks for this new action!

**Pull request merged**